apis/python/src/tiledbsoma/stats.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 TileDB, Inc.
+#
+# Licensed under the MIT License.
+
+import json
+from typing import Any, List, cast
+
+from .pytiledbsoma import tiledbsoma_stats_string
+
+
+def tiledbsoma_stats_json() -> str:
+    """Returns tiledbsoma stats as a JSON string"""
+    # cast is needed for pybind11 things
+    return cast(str, tiledbsoma_stats_string())
+
+
+def tiledbsoma_stats_parsed() -> List[Any]:
```

Comment:
Two very, very small nits:
1. correct me if I'm wrong, but I think you can type the return value more specifically, which would help downstream users.   Something like:  `List[Dict[Literal['counters','timers'], Dict[str, float]]`   Not critical, but I'm a big fan of good type hints for IDE users.  API UX.
2. Also not a big deal - but I wonder if the name wouldn't be better as `tiledbsoma_stats_aspy` or some such?  The word `parsed` doesn't really convey the "return as a python object"

Reply:
Awesome stuff, thanks @bkmartinjr !! :)